src/wp-includes/formatting.php

```diff
@@ -3307,7 +3307,21 @@ function translate_smiley( $matches ) {
 	 */
 	$src_url = apply_filters( 'smilies_src', includes_url( "images/smilies/$img" ), $img, site_url() );
 
-	return sprintf( '<img src="%s" alt="%s" class="wp-smiley" style="height: 1em; max-height: 1em;" />', esc_url( $src_url ), esc_attr( $smiley ) );
+	$html = sprintf( '<img src="%s" alt="%s" class="wp-smiley" style="height: 1em; max-height: 1em;" />', esc_url( $src_url ), esc_attr( $smiley ) );
+
+	/**
+	 * Filters the Smiley image markup.
+	 *
+	 * @since 5.4.0
+	 *
+	 * @param string $html       HTML markup for the smiley image.
+	 * @param string $smiley_url URL for the smiley image.
+	 * @param string $img        Filename for the smiley image.
+	 * @param string $site_url   Site URL, as returned by site_url().
+	 */
+	$html = apply_filters( 'smilies_html', $html, $src_url, $img, site_url() );
```

Comment:
As noted, I don't change the priority on `the_content` is a good idea. It could break content added in filters. We could keep the filter level below 8/9. Making this filter required.

Reply:
Same here, I don't think we should introduce yet another filter here. Why not keep it simple like you did in https://github.com/WordPress/wordpress-develop/pull/123/commits/7d8ff7dd7e6bb39086e198043c9b045f0b177025#diff-c7bcebadc772c3c9048583c515902058R3299?

Reply:
Everything should run through `wp_lazy_load_content_media` as it allow for the second param on that filter to be set, giving developers more context / control.